src/library/entities/Library.java

```diff
@@ -84,153 +86,155 @@ public static synchronized void SaVe() {
-        if (CaTaLoG.containsKey(ItEm_Id)) 
-            return CaTaLoG.get(ItEm_Id);        
+    public Item getItem(long itemId) {
+        if (catalog.containsKey(itemId)) {
+            return catalog.get(itemId);
+        }
         return null;
     }
 
     
-    public int gEt_LoAn_LiMiT() {
-        return lOaNlImIt;
+    public int getLoanLimit() {
+        return LOAN_LIMIT;
     }
 
     
-    public boolean cAn_PaTrOn_BoRrOw(Patron PaTrOn) {        
-        if (PaTrOn.gEt_nUmBeR_Of_CuRrEnT_LoAnS() == lOaNlImIt ) 
+    public boolean canPatronBorrow(Patron patron) {
+        if (patron.gEt_nUmBeR_Of_CuRrEnT_LoAnS() == LOAN_LIMIT) {
             return false;
+        }
                 
-        if (PaTrOn.FiNeS_OwEd() >= maxFinesOwed) 
+        if (patron.FiNeS_OwEd() >= MAX_FINES_OWED) {
             return false;
+        }
                 
-        for (Loan loan : PaTrOn.GeT_LoAnS()) 
-            if (loan.Is_OvEr_DuE()) 
+        for (Loan loan : patron.GeT_LoAnS()) {
+            if (loan.Is_OvEr_DuE()) {
                 return false;
-            
+            }
+        }
         return true;
     }
 
     
-    public int gEt_NuMbEr_Of_LoAnS_ReMaInInG_FoR_PaTrOn(Patron pAtRoN) {        
-        return lOaNlImIt - pAtRoN.gEt_nUmBeR_Of_CuRrEnT_LoAnS();
+    public int getNumberOfLoansRemainingForPatron(Patron patron) {
+        return LOAN_LIMIT - patron.gEt_nUmBeR_Of_CuRrEnT_LoAnS();
     }
 
     
-    public Loan iSsUe_LoAn(Item iTeM, Patron pAtRoN) {
-        Date dueDate = Calendar.GeTiNsTaNcE().GeTdUeDaTe(loanPeriod);
-        Loan loan = new Loan(gEt_NeXt_LoAn_Id(), iTeM, pAtRoN, dueDate);
-        pAtRoN.TaKe_OuT_LoAn(loan);
-        iTeM.TaKeOuT();
-        LoAnS.put(loan.GeT_Id(), loan);
-        CuRrEnT_LoAnS.put(iTeM.GeTiD(), loan);
+    public Loan issueLoan(Item item, Patron patron) {
+        Date dueDate = Calendar.GeTiNsTaNcE().GeTdUeDaTe(LOAN_PERIOD);
+        Loan loan = new Loan(getNextLoanId(), item, patron, dueDate);
+        patron.TaKe_OuT_LoAn(loan);
+        item.TaKeOuT();
+        loans.put(loan.GeT_Id(), loan);
+        currentLoans.put(item.GeTiD(), loan);
         return loan;
     }
     
     
-    public Loan GeT_LoAn_By_ItEm_Id(long ITem_ID) {
-        if (CuRrEnT_LoAnS.containsKey(ITem_ID)) 
-            return CuRrEnT_LoAnS.get(ITem_ID);
-        
+    public Loan getLoanByItemId(long itemId) {
+        if (currentLoans.containsKey(itemId)) {
+            return currentLoans.get(itemId);
+        }
         return null;
     }
 
     
-    public double CaLcUlAtE_OvEr_DuE_FiNe(Loan LoAn) {
-        if (LoAn.Is_OvEr_DuE()) {
-            long DaYs_OvEr_DuE = Calendar.GeTiNsTaNcE().GeTDaYsDiFfErEnCe(LoAn.GeT_DuE_DaTe());
-            double fInE = DaYs_OvEr_DuE * FiNe_PeR_DaY;
-            return fInE;
+    public double calculateOverDueFine(Loan loan) {
+        if (loan.Is_OvEr_DuE()) {
+            long daysOverDue = Calendar.GeTiNsTaNcE().GeTDaYsDiFfErEnCe(loan.GeT_DuE_DaTe());

```

Comment:
See https://github.com/elliotmitchum/ITC205/pull/35/files#r934038851

Reply:
commit -m"refining calculateOverDueFine method to conform with CSG10"
commit:b8ed16c30c6395af72a6ff49619cbf062524335a
@elliotmitchum @haynescrossley